MANIFEST

```diff
@@ -432,6 +433,7 @@ src/memkind_local.c
 src/memkind_log.c
 src/memkind_mem_attributes.c
 src/memkind_memtier.c
+src/memkind_memtier_mtt_allocator.c
```

Comment:
so far, I did not prefix new files with memkind_memtier_ ... I don't think it was mentioned during code review, but we might want to have a consistent naming. What do you think in general ? @bratpiorka ?